src/ResourceManager/Compute/Commands.Compute/Extension/AEM/SetAzureRmVMAEMExtension.cs

```diff
@@ -61,11 +61,18 @@ public class SetAzureRmVMAEMExtension : VirtualMachineExtensionBaseCmdlet
                 Mandatory = false,
                 Position = 2,
                 ValueFromPipelineByPropertyName = false,
-                HelpMessage = "If this parameter is provided, the commandlet will not enable Windows Azure Diagnostics for this virtual machine.")]
+                HelpMessage = "Deprecated - Windows Azure Diagnostics is now disabled by default")]
         public SwitchParameter DisableWAD { get; set; }
 
         [Parameter(
                 Mandatory = false,
+                Position = 2,
+                ValueFromPipelineByPropertyName = false,
+                HelpMessage = "If this parameter is provided, the commandlet will enable Windows Azure Diagnostics for this virtual machine.")]
+        public SwitchParameter EnableWAD { get; set; }
```

Comment:
@MSSedusch currently `EnableWAD` and `DisableWAD` can be called in the same command. Can you create two parameter sets, one that contains `EnableWAD` and one that contains `DisableWAD`, so that this scenario is not possible?